automatic/webstorm/readme.md

```diff
@@ -0,0 +1,18 @@
+WebStorm — The smartest JavaScript IDE. Lightweight yet powerful IDE,
```

Comment:
Would be preferable to have the icon and title at the top here:
for instance:
`# [<img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/8edfc93894e9bdd11deb36d1cbbfe8a47a7b013a/icons/webstorm.png" height="48" width="48" /> webstorm](https://chocolatey.org/packages/webstorm)`

In this case you would also need to add `-SkipFirst 1` for the `Set-DescriptionFromReadme` in the `update.ps1` file.